pytransform3d/transform_manager/_temporal_transform_manager.py

```diff
@@ -44,14 +48,84 @@ class StaticTransform(TimeVaryingTransform):
     def __init__(self, A2B):
         self._A2B = A2B
 
-    def as_matrix(self, time):
+    def as_matrix(self, query_time):
         return self._A2B
 
     def check_transforms(self):
         self._A2B = check_transform(self._A2B)
         return self
 
 
+class NumpyTimeseriesTransform(TimeVaryingTransform):
+    """Transformation sequence, represented in a numpy array.
+    The interpolation is computed using screw linear interpolation (ScLERP)
```

Comment:
There should be an empty line before this line

Reply:
Added